code/controllers/subsystem/dbcore.dm

```diff
@@ -170,17 +170,26 @@ SUBSYSTEM_DEF(dbcore)
 	connection = SSdbcore.connection
 
 /datum/controller/subsystem/dbcore/Shutdown()
+	to_chat(world, span_boldannounce("Clearing DB queries standby:[length(queries_standby)] active: [length(queries_active)] all: [length(all_queries)]"))
 	//This is as close as we can get to the true round end before Disconnect() without changing where it's called, defeating the reason this is a subsystem
 	if(SSdbcore.Connect())
+		//Execute all waiting queries
 		for(var/datum/db_query/query in queries_standby)
-			run_query(query)
+			run_query_sync(query)
+			qdel(query)
```

Comment:
i'm not sure it makes sense to do this, because somethign else out there might be holding a ref to this query too.

But I also can't think of what would still be out there trying to run queries during world shutdown.

only thing i can think off is stuff happening when players connect.